Package@swift-5.3.swift

```diff
@@ -89,16 +89,16 @@ let package = Package(
             path: "AppCenterDistribute/AppCenterDistribute",
             exclude: ["Support"],
             resources: [
-                .copy("AppCenterDistributeResources.bundle"),
+                .process("Resources/AppCenterDistribute.strings"),
             ],
             cSettings: [
                 .headerSearchPath("**"),
                 .headerSearchPath("../../AppCenter/AppCenter/**"),
             ],
             linkerSettings: [
                 .linkedFramework("Foundation"),
-                .linkedFramework("SafariServices"),
-                .linkedFramework("UIKit"),
+                .linkedFramework("SafariServices", .when(platforms: [.iOS])),
+                .linkedFramework("UIKit", .when(platforms: [.iOS])),
```

Comment:
AppCenterDistribute module itself works only for iOS. Limit module itself instead of dependencies

Reply:
Not sure that it is possible, because I didn't found any clear guides for it.

Reply:
Please add clear error for that case then